src/Admin/AdminExtensionInterface.php

```diff
@@ -70,10 +54,7 @@ public function configureTabMenu(
      */
     public function validate(AdminInterface $admin, ErrorElement $errorElement, $object);
 
-    /**
-     * @param string $context
-     */
-    public function configureQuery(AdminInterface $admin, ProxyQueryInterface $query, $context = 'list');
+    public function configureQuery(AdminInterface $admin, ProxyQueryInterface $query);
```

Comment:
Could we add return declarations for these methods?

Reply:
I was talking about all the methods in the interface. Although this could be done in a separate task if you think that way will be better.

Reply:
I also realised that the `$context` parameter was there 😬

As you said I think it's better to do it in another PR and leave this about removing stuff for the sake of reviewing.